### PR TITLE
Communication codes w/reorg

### PR DESCRIPTION
**Pull request by kawilliams**
- Added new codes related to communication in the reorganized codes.md (based on Jamboard)
- Counted number of people who added annotations
    - Was only 2-3. I wish I had more consistently noted when people used interactions with their pen/finger/hand, those were more prevalent
- Added code about the language used when asked to "Describe your sketch"

### CODE REVIEW

File: codes.md [hunk 1]
```diff
@@ -589,6 +594,26 @@ KATY/ALEX: Is this a sub-code of depiction?
...
+  - Named a data abstraction or idiom: 9, 16, 17, 20, 23, 26, 28, 29
+
+    - NOTE: they did not always correctly name the abstraction or idiom that they used
+
+  - Only gave a visual description (usually of the icons or marks used): 6, 7, 10, 11, 12, 13, 15, 19, 21, 22, 24, 25, 30, 31, 33
+
+  - Did not describe the drawing (restated the dataset): 8, 14, 18, 27, 32
+
+  - This could have implications for accessibility -- when writing alt text for graphs or when presenting to visually impaired audiences.
+
```
Comment (kisaacs) [inline codes.md:621]: Could it be these two codes should be combined into one regarding how descriptions of drawings focused on data solely, marks solely, or a blend with the visual idiom -- is it purely the data, purely the drawing, or is it the data abstraction and essentially a verbal "key". (Is it that the ones who described the idioms often also described the marks?)

Comment (kawilliams) [reply codes.md:621]: We can combine them into 1 code -- they're the same code, I separated them out for my own clarity as I was building it. 

The ones who provided the data abstraction typically stated it in the first or last sentence (e.g. Session 20 with something like "the power stations are nodes, I have connections... A graph, basically.") _and_ they typically included a verbal "key" to describe the marks. 

The ones who only described the marks (a verbal key) did so in a range of detail. Some methodically explained every mark, some started to go through all of them but got distracted.

~~I'll change the code to:~~ (Now I'm wondering if we have 2 axes of detail)
### The "Describe your sketch" prompt yielded a range in the detail of the descriptions, from recitation of the data points to the data abstraction, and from individual icons to descriptions of the overall visualization

Comment (kawilliams) [reply codes.md:621]: Might be overcomplicating it by making it 2-D. Maybe just "individual data points --> individual icons --> relations of icons / positions of icons --> data abstraction"

Comment (kawilliams) [reply codes.md:621]: We decided to go with the single axis levels of details. Codes were combined.

**Pull request merged**